README.md

```diff
@@ -16,6 +16,17 @@ xattr -d com.apple.quarantine ./path/to/gof5_darwin
 
 On FreeBSD only `wireguard` or `ppp` (a wrapper around the ppp binary) drivers work. However the `wireguard` driver requires a [FreeBSD patch](freebsd.patch) applied before the compilation.
 
+## Linux
+
+Wireguard comes on most kernels, so you only need to install any dependencies, if you want to `ppp`
```

Comment:
@alfredopalhares wireguard go package is used only to setup the `tun` interface, as well as [water](https://github.com/songgao/water) go package. It doesn't use the wireguard protocol, so it doesn't require any kernel modules. The actual F5 data protocol is an encapsulation over HTTPS.
In case, when pppd (or ppp in FreeBSD) is used, gof5 doesn't setup a tun interface, but just establishes an HTTPS connection and forwards the encapsulated traffic to the pppd binary and pppd creates the tunnel etc.

Reply:
Understood. I omitted the Wireguard reference.